main.rb

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+require_relative 'app'
+def main
+  app = App.new
+  app.run
+end
+
+main
```

Comment:
Currently, your application is running an infinite loop for all the options other than option 7. Kindly fix this error.
Here is a screenshot of the problem ⬇️. 
You might have forgotten to call a certain function in the app.rb file inside function `run` 😊
![image](https://user-images.githubusercontent.com/33268927/194696952-5122abd5-ad43-4f4b-b27c-5d50b624a1ce.png)

Reply:
@muneebulrehman Thank you for pointing this out, the missing method call has been added, and now it works as expected :heavy_check_mark: 

![image](https://user-images.githubusercontent.com/92395484/194717312-a84fd66a-1337-4140-85a1-5b505394c17d.png)